<launch>
  <arg name="image" default="/downward_cam/camera/image" />
  <arg name="projection" default="/paramatrix" />
  
  <node pkg="jsk_mbzirc_tasks" type="uav_detect_landing_region_node"
	name="uav_detect_landing_region" output="screen" >
    <remap from="input_image" to="$(arg image)" />
    <!-- <remap from="input_mask" to="$(arg image)" /> -->
    <remap from="input_mask" to="/track_region_mapping/output/track_mask" />
    <remap from="input_proj_mat" to="$(arg projection)" />
  </node>

  <node pkg="jsk_mbzirc_tasks" type="non_maximum_suppression.py"
	name="non_maximum_suppression" output="screen" >
    <remap from="non_maximum_suppression" to="non_maximum_suppression" />
  </node>
  
  <node pkg="jsk_mbzirc_tasks" type="track_region_mapping.py"
	name="track_region_mapping" output="screen" >
  </node>

</launch>
